File: redsheet-portal/src/app/database-models/lite-our-team-member.ts
import { required, maxLength, range, nested } from '@rx/annotations';
import { LiteProjectBackground,  vLiteOurTeamMemberRecord  } from './'
export class LiteOurTeamMember {
    constructor(liteOurTeamMember?: LiteOurTeamMember  | vLiteOurTeamMemberRecord )  {
        let properties = [ "liteOurTeamMemberId", "liteOurTeamMemberName", "personalityId", "position", "liteProjectBackgroundId",];
        for (let property of properties)
            if (liteOurTeamMember && liteOurTeamMember[property])
                this[property] = liteOurTeamMember[property];
    }
 
	liteOurTeamMemberId : number =   0 ;
 
    @required()
    @maxLength(50)
	liteOurTeamMemberName : string =   undefined;
 
    @range(1,2147483647)
	personalityId : number =   undefined;
 
    @required()
    @maxLength(50)
	position : string =   undefined;
 
    @range(0,2147483647)
	liteProjectBackgroundId : number =   undefined;
	liteProjectBackground : LiteProjectBackground  ;


}

File: RedSheet.DbEntities/Models/LiteOurTeamMember.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("LiteOurTeamMembers", Schema = "dbo")]
	[Serializable]
	public partial class LiteOurTeamMember  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int LiteOurTeamMemberId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string LiteOurTeamMemberName { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int PersonalityId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string Position { get; set; } 
 
	[RelationshipTableAttribue("LiteProjectBackgrounds","dbo")]
	//Relationships
	public int LiteProjectBackgroundId { get; set; } 
 
	[ForeignKey("LiteProjectBackgroundId")]
	public LiteProjectBackground LiteProjectBackground { get; set; } 

	public LiteOurTeamMember()
	{
    }

    }
}

